src/shared/components/progressIndicator/ProgressIndicator.tsx

```diff
@@ -59,10 +63,13 @@ export default class ProgressIndicator extends React.Component<IProgressIndicato
             }
         }
         return (
-            <span className={classnames(styles["item-row"], {[styles["item-row-not-first"]]:(index > 0), [styles["not-invoked-item"]]:notInvoked})}>
-            {item.label}
+            <span
+                className={classnames(styles["item-row"], {[styles["item-row-not-first"]]:(index > 0), [styles["not-invoked-item"]]:notInvoked})}
```

Comment:
you could accomplish this with nth-child?

Reply:
thats true but i do feel that selector is a bit hacky/coupling view structure w logic. better IMO to just tag it manually. what do you think?

Reply:
hmmm.  if it's just style, it seems opposite of hacky to me.  in there words, logic/function world doesn't care at all about the fact that the row is first.  it's just a style thing so it should be unobtrusively applied via selector.